src/css/style.css

```diff
@@ -0,0 +1,71 @@
+.outerDiv {
+  background-image: url(../hero-bg.jpg);
+  width: 100%;
+  height: 700px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  text-align: center;
+  padding: 20px;
+}
+
+.inner-container {
+  background-color: rgba(0, 0, 0, 0.479);
+  margin: 100px;
+  padding: 20px;
+  color: whitesmoke;
+}
+
+.navCont {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  top: 0;
+  left: 0;
+  width: 100%;
+  position: fixed;
+  height: 100px;
+  background-color: yellowgreen;
+}
+
+li {
+  display: inline-block;
+  color: whitesmoke;
+  list-style: none;
+  padding-left: 20px;
+  padding-right: 20px;
+  font-size: 20px;
+  font-weight: bold;
+  border: #a1a1a1;
+  float: left;
+  -webkit-transition: margin 0.5s ease-out;
+  -moz-transition: margin 0.5s ease-out;
+  -o-transition: margin 0.5s ease-out;
+}
+
+li:hover {
+  cursor: pointer;
+  margin-top: 5px;
+}
+
+.logo {
+  display: block;
+  width: 70px;
+  height: 70px;
+}
+
+.foodContainer {
+  width: 20rem;
+  max-height: 300px !important;
```

Comment:
- [ ] Kindly avoid using the !important rule in css as it doesn't meet the best practices requirements. Please remove it.